inc/mailcollector.class.php

```diff
@@ -1071,52 +1053,71 @@ function decodeMimeString($mimeStr, $inputCharset = 'utf-8', $targetCharset = 'u
 
 
    /**
-     * Connect to the mail box
-   **/
+    * Connect to the mail box
+    *
+    * @return void
+    */
    function connect() {
 
       if ($this->fields['use_kerberos']) {
-         $this->marubox = @imap_open($this->fields['host'], $this->fields['login'],
-                                     Toolbox::decrypt($this->fields['passwd'], GLPIKEY),
-                                     CL_EXPUNGE, 1);
+         $this->marubox = imap_open(
+            $this->fields['host'],
+            $this->fields['login'],
+            Toolbox::decrypt($this->fields['passwd'], GLPIKEY),
+            CL_EXPUNGE,
+            1
+         );
       } else {
-         $try_options = [['DISABLE_AUTHENTICATOR' => 'GSSAPI'],
-                              ['DISABLE_AUTHENTICATOR' => 'PLAIN']];
+         $try_options = [
+            ['DISABLE_AUTHENTICATOR' => 'GSSAPI'],
+            ['DISABLE_AUTHENTICATOR' => 'PLAIN']
+         ];
          foreach ($try_options as $option) {
-            $this->marubox = @imap_open($this->fields['host'], $this->fields['login'],
-                                        Toolbox::decrypt($this->fields['passwd'], GLPIKEY),
-                                        CL_EXPUNGE, 1, $option);
+            $this->marubox = imap_open(
+               $this->fields['host'],
+               $this->fields['login'],
+               Toolbox::decrypt($this->fields['passwd'], GLPIKEY),
+               CL_EXPUNGE,
+               1,
+               $option
+            ) or die(imap_last_error());
```

Comment:
Looks like you are in a loop to try multiple auth options. The `die` is not compatible with multiple tries.

Reply:
Fixed in 9. PR